internal/driver/glfw/window.go

```diff
@@ -635,6 +644,13 @@ func (w *window) mouseClicked(_ *glfw.Window, btn glfw.MouseButton, action glfw.
 		w.mouseButton = 0
 	}
 
+git	// we cannot switch here as objects may respond to multiple cases
```

Comment:
typo I'm guessing

Reply:
With two screens, it is not easy to know which window is focused...